release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -85,6 +85,13 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-security"]
 === Security and compliance
 
+[id="ocp-4-11-security-oauth-server-audit-log"]
+==== OAuth server events are now logged in the audit logs
+
+OAuth server events, including failed login attempts, are now logged at the metadata level in the audit logs.
```

Comment:
Similarly (this one is a bit clunky flipped around tho imo :thinking:):

```suggestion
The audit logs now log OAuth server events, including failed login attempts, at the metadata level.
```

Reply:
I would emphasize that login events are annotated. As it is not an integral part of the audit event, but added on top.
As the login failures are the really new thing, we could emphasize this fact in a follow up sentence. WDYT?

> OAuth server events, annotated with login events, are now logged at the metadata level in the audit logs. The login events include failed attempts.